src/components/shared/customFooter.js

```diff
@@ -28,7 +28,7 @@ const DefaultFooter = ({logos, text, widgets}) => (
       <div className="col-md-3 content-area">
       {logos.map(logo => (
         <GatsbyImage
-                        image={logo.relationships.field_media_image.localFile.childImageSharp.gatsbyImageData}
+                        image={logo.relationships.field_media_image.gatsbyImage}
```

Comment:
Could use getImage here and remove the .gatsbyImage. That way if the object changes in future, we can just update the query. If you go this route, you'd also need to add getImage to the GatsbyImage imports above.

Reply:
Done